lib/src/base-styles/_nx-pagination.scss

```diff
@@ -66,21 +71,22 @@
   }
 
   &:active {
-    background-color: var(--nx-swatch-grey-97);
-    color: var(--nx-color-text);
+    background-color: var(--nx-color-pagination-background-active);
+    color: var(--nx-color-text-stark);
     outline: none;
   }
 
   &.selected {
     &, &:hover {
-      background-color: var(--nx-swatch-blue-40);
+      background-color: var(--nx-color-primary-button-background);
       border-color: transparent;
-      color: var(--nx-color-component-background);
+      color: var(--nx-color-pagination-focus-selected);
       cursor: default;
+      font-weight: 600;
     }
-    
+
     &:focus {
-      outline: 2px solid var(--nx-color-component-background);
+      outline: 2px solid var(--nx-color-pagination-focus-selected);
       outline-offset: -4px;
```

Comment:
```suggestion
      outline-offset: calc(0px - var(--nx-focus-outline-width) - var(--nx-focus-outline-offset));
```

Reply:
> 

It doesn't look like the `--nx-focus-outline-offset` variable is defined anywhere. However, `NxCheckbox.scss` seems to use it ❓

Reply:
> > > > 
> > > 
> > > 
> > > It doesn't look like the `--nx-focus-outline-offset` variable is defined anywhere. However, `NxCheckbox.scss` seems to use it ❓
> > 
> > 
> > Oooh that's interesting re: NxCheckbox. So I think what Ross is referring to is we made variables during the NxButton ticket to standardize some of the consistent outline pixel values. However, the variable is called `--nx-focus-outline-inset` as opposed to `---nx-focus-outline-offset`. It seemed to make sense to me at the time (since the outline is technically inside not outside), but maybe not the best variable name. That renaming however could be tackled in RSC-1554 (https://issues.sonatype.org/browse/RSC-1554)
> 
> Changed to use the `--nx-focus-outline-inset` variable here: [7f0e9bb](https://github.com/sonatype/sonatype-react-shared-components/commit/7f0e9bb48f08faa39ce2af380ab159197389eca5)

Do you want to make a ticket to fix the problem in `NxCheckbox.scss`? I just tried updating it to the correct variable, which completely threw off the focus border, so that will need to be fixed.
![Screenshot 2023-04-19 at 12 59 57 PM](https://user-images.githubusercontent.com/89862727/233147434-00c5ddeb-87a6-4712-a3ad-9bb6507c1b6d.png)

Reply:
> -nx-focus-outline-inset

Ticket here: https://issues.sonatype.org/browse/RSC-1649